homeassistant/components/input_number/services.yaml

```diff
@@ -30,6 +30,24 @@ set_value:
           step: 0.001
           mode: box
 
+add:
+  name: Add
+  description: Increase the value of an input number entity by specified amount.
+  target:
+    entity:
+      domain: input_number
+  fields:
+    amount:
+      name: Amount
+      description: The amount to increase the current value by.
+      required: true
+      selector:
+        number:
+          min: 0
```

Comment:
For this to be useful in automations and scripts we should allow both negative and positive adjustments.
We could call the new service "step" instead, similar to `brightness_step` for lights, or "adjust", and allow both negative and positive values.

Reply:
I just had a thought. What if instead of adding a new service we add an optional argument to `increment` and `decrement` services that would override `step` from configuration? Would it be too confusing?